concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -83,6 +83,7 @@ resources:
     source:
       uri: https://github.com/alphagov/paas-graphite-statsd-boshrelease.git
       tag_filter: {{cf_graphite_version}}
+      branch: 118272895_bosh20
 
```

Comment:
> Do we need to specify `branch` if we're pinning the commit with `tag_filter` too?

Yes. @saliceti investigated and the git resource looks at the `branch`, only returning tags that match `tag_filter`.